### DISCUSSION TITLE
Rencana Rewrite pejuangkode.com

### DISCUSSION THREAD

**Discussion by zainfathoni** [pejuang-kode/pejuangkode.com#1] [General]
Berhubung [Netlify sudah full mendukung Next.js](https://www.netlify.com/blog/2020/12/07/announcing-one-click-install-next.js-build-plugin-on-netlify/) juga, saya jadi terpikir untuk rewrite website ini dari GatsbyJS ke Next.js dengan memanfaatkan ilmu yg saya pelajari dari [EpicReact.dev](https://epicreact.dev).

# Usulan Fitur

- [Usulan dari Mas @ekaone](https://github.com/zainfathoni/pejuangkode/discussions/16#discussioncomment-214029)

**Comment by ekaone** [comment]
Rangkuman:

- Framework [Next.js](https://nextjs.org/)
- Environment [Netlify](https://www.netlify.com/)
- Design System [Tailwindcss](https://tailwindcss.com/)
- Codebases JS / ~TS~
- Halaman Informasi Umum
- Halaman Infomasi Komunitas Frontend Indonesia
- Halaman FAQ
- Halaman e-learning (berbentuk halaman dokumentasi), mungkin dibuat sub-domain (beda project)

Silahkan ditambahkan untuk ide-ide lainnya 👌

**Reply by zainfathoni** [reply]
Apa mau nyobain pake TS Mas @ekaone ?
Saya belum pernah setup TS di NextJS sih, tapi kayaknya kita bisa ngikutin [panduan di sini](https://nextjs.org/learn/excel/typescript).

**Reply by ekaone** [reply]
Boleh neh Mas, mumpung masih seger coba TS 😃
